### ISSUE TITLE
Add Unit Tests with 100% Code Coverage for `repos.js`

### ISSUE THREAD

**Issue by Furchin** [builtforme/CodeBnb#20]
Right now there are none.

**Comment by h93xV2** [comment]
I'm going to start working on this.

**Comment by h93xV2** [comment]
Seems like I bit off more than I could chew with this one. If it's alright, I'd like to switch my claim to #18 since I know I can get that one done.

**Comment by mcclayton** [comment]
Hi @h93xV2, I already implemented 100% code coverage for https://github.com/builtforme/CodeBnb/issues/18 here: https://github.com/builtforme/CodeBnb/pull/22

However, I'm sure there are a number of patterns in my PR that would help in creating tests for repos.js. Keep in mind that you don't necessarily have to get 100% coverage in a single PR. You can work incrementally towards it in a few PRs and others can help you that way too. Hope that helps!

**Comment by h93xV2** [comment]
@mcclayton Okay, sounds great. You're PR is written well, so I may end up copying some of the patterns.

**Comment by Furchin** [comment]
Yep, I'll gladly consider PRs which improve code coverage, and only resolve the issue once coverage reaches 100% (and as noted, some branch exceptions may apply as it can indeed be difficult to hit 100% and the effort may not be worth it for some small edge scenarios).
